src/backend/joanie/core/api/admin.py

```diff
@@ -221,6 +221,8 @@ def create(self, request, *args, **kwargs):
         data = request.data
```

Comment:
This override makes me wonder if it would be possible to do the same but calling super() and adding only the handling of the course_runs field in the override.